Force C local when generating the documentation
Without the C locale, the generated file can already have translations in the current locale...
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -18,7 +18,7 @@
 	touch hg.1.txt
 
 hg.1.gendoc.txt: $(GENDOC)
-	${PYTHON} gendoc.py > $@.tmp
+	env LC_ALL=C ${PYTHON} gendoc.py > $@.tmp
 	mv $@.tmp $@
 
 %: %.txt common.txt
